apps/openmw/mwworld/weather.cpp

```diff
@@ -660,6 +660,10 @@ void WeatherManager::update(float duration, bool paused)
     mWindSpeed = mResult.mWindSpeed;
     mIsStorm = mResult.mIsStorm;
 
+    // For some reason Ash Storm is not considered as a precipitation weather in game
```

Comment:
Well, there's not much difference between a blight storm and an ash storm mechanically, so why even have a special case for ash storms - merely for replicating the original perhaps buggy behavior?

Reply:
> merely for replicating the original perhaps buggy behavior?

Yes. But I do not know if it is buggy or intended behaviour, so I decided to replicate it for now.
If we do not want this case, we just can remove it.